Installer/source/main.cpp

```diff
@@ -236,6 +236,7 @@ bool CompareSha1Hash(u8 *Data1,u32 Data1_Size,u8 *Data2,u32 Data2_Size)
 	}
 	unsigned int FileHash_D1[5];
 	unsigned int FileHash_D2[5];
+	memset(FileHash_D1,0x00,5);

```

Comment:
these sizes are better to be changed to sizeof(unsigned int)*5, with both FileHash_D1 & D2
also, maybe move the comment to above FileHash explaining that they are set manually to have them have an initial value that differs so checks fail by default